Document/0x04c-Tampering-and-Reverse-Engineering.md

```diff
@@ -175,6 +175,7 @@ For more complicated tasks, such as de-obfuscating heavily obfuscated binaries,
 
 As always in hacking, the anything-goes rule applies: simply use whatever is most efficient. Every binary is different, and all reverse engineers have their own style. Often, the best way to achieve your goal is to combine approaches (such as emulator-based tracing and symbolic execution). To get started, pick a good disassembler and/or reverse engineering framework, then get comfortable with their particular features and extension APIs. Ultimately, the best way to get better is to get hands-on experience.
 
+
```

Comment:
Hi @su-vikas, thank you very much for this PR again! I see quiet a few markdown linting errors being added over time. These can cause issues when we generate the various formats of the MSTG when we release it. Issues that often happen:
- spaces just before a newline
- extra newlines (such as these)

I love the content that you write, which is why I wonder: can you help us out by using a tool like visual studio code with markdown linting (see step 7 in https://github.com/OWASP/owasp-mstg/blob/master/CONTRIBUTING.md#how-to-set-up-your-contributor-environment) . Or run `./tools/Apply_Linter_Check.sh` and fix its feedback?

Reply:
Noted. Will run linter from now on.

Reply:
Thanks!